compiler/tools/fpp-to-cpp/test/component/ActiveStateMachines_S1.hpp

```diff
@@ -0,0 +1,96 @@
+
+// ======================================================================
+// \title  ActiveStateMachines_S1.h
+// \author Auto-generated
+// \brief  header file for state machine ActiveStateMachines_S1
+//
+// ======================================================================
+           
+#ifndef ACTIVESTATEMACHINES_S1_H_
+#define ACTIVESTATEMACHINES_S1_H_
+                                
+#include <Fw/SMSignal/SMSignalBuffer.hpp>
```

Comment:
For consistency with the rest of F Prime, I think the path should be `Fw/Sm/SmSignalBuffer.hpp`. Compare `Fw/Com/ComPacket.hpp`, `Fw/Dp/DpContainer.hpp`, etc.

Reply:
Done

Reply:
Looks good!